cypress/cypress/e2e/03_submariner_deploy.cy.js

```diff
@@ -51,7 +54,9 @@ describe('submariner - Deployment validation', {
         cy.get('#globalist-enable').click({ force: true })
         cy.get('button').contains('Next').click()
 
-        cy.get('.pf-c-wizard__nav-list').eq(1).children().each(() => {
+        cy.get('.pf-c-wizard__nav-list').eq(1).children().each(($children) => {
+            childrenName = $children.text()
```

Comment:
@NogaTal98 
What is the meaning of "children" and "childrenName" variable?
It's quite misleading.